### ISSUE TITLE
Screenshot shows boxes instead of Korean characters

### ISSUE THREAD

**Issue by ryanckulp** [vercel/og-image#108]
hi folks,

great utility, deploying to a site with thousands of pages. my challenge is: Korean (한글) characters aren't working.

this is similar to my issue (Hindi language):
https://github.com/puppeteer/puppeteer/issues/5311

here's what i'm seeing in my Zeit-deployed production instance, w/ text `"경찰" World`:

<img width="730" alt="Screen Shot 2020-04-13 at 11 46 07 PM" src="https://user-images.githubusercontent.com/3083888/79131751-14035680-7de4-11ea-848c-5d994c667ed2.png">

**but in localhost:3000 it works great**!

<img width="1266" alt="Screen Shot 2020-04-13 at 11 44 41 PM" src="https://user-images.githubusercontent.com/3083888/79131768-1bc2fb00-7de4-11ea-9619-6f21e0d9d7d9.png">

i've tried changing:
* UTF
* lang
* fonts (e.g. Arial vs Inter)
* hard-coding Korean characters into the template e.g. `<div class="heading">경찰 test` (inside `template.js`)

i tried these measures to ensure local sanitizers are doing anything funky.

every one of my tests yields the same: works in localhost, doesn't in production. thanks for reading.

**Comment by styfle** [comment]
Related to #70

**Comment by ryanckulp** [comment]
update.. after 10 hours i'm almost. there.

**tldr** -- replaced puppateer w/ selenium-webdriver.

one question about Zeit server config: what's the executable path i should be using for Chrome, in prod?

tried: `/tmp/chromium` // `/usr/local/bin/chromedriver`

the latter works locally but not in production, says "PATH NOT FOUND."

**Comment by styfle** [comment]
ZEIT Now doesn't ship Chrome so you would have to consult the docs for selenium-webdriver to find out where it installs Chrome for Linux. If you can set a path, I would suggest `/tmp` because that is the only writable directory on the filesystem.

**Comment by AntwanSherif** [comment]
@ryanckulp Could you share with me how did you solve this issue? Also it it applicable for Arabic language for example?

**Comment by r0mflip** [comment]
Not all fonts support all scripts. Some of the suggested ones would be to use from the Noto family, but even then it doesn't include CJK, Devanagri, Arabic, ...

If https://github.com/vercel/og-image/issues/70 works then it would be awesome that we can the required font and tweak the CSS a bit to support the required script. IMHO all-in-one solution might be a far fetch.

**Comment by hoosan** [comment]
I have run into the same "tofu" issue with Japanese characters and got stuck for a while.
As @r0mflip mentioned, adding the Noto family was the way to go. 
I share the procedure in case someone is in the same trouble.

1. Put a font file (e.g. *.otf) in "_fonts" directory.
![image](https://user-images.githubusercontent.com/40290137/87226597-d6589e80-c3cf-11ea-90d7-4fb6ce039616.png)

2. Read the font file in template.ts

```const noto = readFileSync(`${__dirname}/../_fonts/NotoSansJP-Black.otf`).toString('base64');```

![image](https://user-images.githubusercontent.com/40290137/87226625-fee09880-c3cf-11ea-8506-b68a1f22f76c.png)

3. Inject it to CSS

```
    @font-face {
        font-family: 'Noto Sans Japanese';
        font-style: normal;
        font-weight: normal;
        src: url(data:font/otf;charset=utf-8;base64,${noto}) format('opentype');
    }
```

4. Add its font family name to the heading class

```
    .heading {
        font-family: 'Inter', 'Noto Sans Japanese', sans-serif;
        font-size: ${sanitizeHtml(fontSize)};
        font-style: normal;
        color: ${foreground};
        line-height: 1.8;
    }
```

That's it. It works fine *most of the time*.

![image](https://user-images.githubusercontent.com/40290137/87226856-94c8f300-c3d1-11ea-804c-fb0a1764842c.png)

An unresolved issue is that the some characters are still not visible. 

![image](https://user-images.githubusercontent.com/40290137/87226950-45cf8d80-c3d2-11ea-9816-bba996e77e51.png)

But If I *double* the character ("あ" in this case), the problem has gone.

![image](https://user-images.githubusercontent.com/40290137/87227085-52081a80-c3d3-11ea-8af3-6d56cb49e800.png)

Dunno why.. I will try to add some more font files and see how it goes.

**Comment by tomy0000000** [comment]
I want to point out that even though most of us can get around with this problem by injecting Noto fonts in to HTML templates for now, the true issue here is actually lies within the `chrome-aws-lambda`.

I did some experiment:
- Try running on my local dev: worked -> which means the currently provided fonts is fine.
- Deployed on Vercel: failed -> Something happened on the cloud went wrong.
- Deployed another instance with `OG_HTML_DEBUG=1` envs: worked -> The render html can properly display, which leaves the only possibility to the screenshot function.

**Comment by leerob** [comment]
Closing in favor of https://github.com/vercel/og-image/issues/226
